cron/cleanTimeSeries.sql

```diff
@@ -0,0 +1,10 @@
+-- BLOCK clean_time_series
+WITH selected_ids AS (
+    SELECT id
+    FROM time_series
+    WHERE date < now() - make_interval(secs => $retention_period_sec)
```

Comment:
Add `ORDER BY date` (or `id`?) to improve deletion locality? I'm not sure this really matters?

Reply:
I'm not sure either... let's wait until we get #6651 deployed so we can `EXPLAIN` and see how the query planner handles this?

Reply:
Sounds good to me!

Reply:
Although with your new plan to just delete all of them I guess this no longer matters.

Reply:
Agreed!